pkgs/applications/networking/cluster/hetzner-kube/default.nix

```diff
@@ -11,6 +11,13 @@ buildGoModule rec {
     sha256 = "1iqgpmljqx6rhmvsir2675waj78amcfiw08knwvlmavjgpxx2ysw";
   };
 
+  patches = [
+    # Use $HOME instead of the OS user database.
+    # Upstream PR: https://github.com/xetys/hetzner-kube/pull/346
+    # Unfortunately, the PR patch does not apply against release.
+    ./fix-home.patch
```

Comment:
if this has been upstreamed, do you mind using `fetchpatch` to retrieve it?

Reply:
If not please upstream it.

Reply:
I did upstream it (that's the PR link). It's just that the PR against upstream `master` does not apply to the release we're using because of minor conflicts.

Does the comment need improvement?

Reply:
nah, looks fine